amundsen_application/static/js/components/AnnouncementPage/index.tsx

```diff
@@ -52,17 +52,19 @@ export class AnnouncementPage extends React.Component<AnnouncementPageProps> {
   render() {
     return (
       <DocumentTitle title="Announcements - Amundsen">
-        <div className="container announcement-container">
+        <main className="container announcement-container">
           <div className="row">
             <div className="col-xs-12">
-              <h3 id="announcement-header">Announcements</h3>
+              <h1 id="announcement-header" className="h3">
```

Comment:
Yes, you are right, we need to separate what is a styling matter (doing with classes) from what is a content structure matter (marked by the tags).

In general terms, all documents need to have at least one <h1>, and this is it for this page.

Reply:
> In general terms, all documents need to have at least one 'h1'

For what reason? I don't know the full context of where that recommendation comes from, but I think about how if our designs don't call for an `h1, then we wouldn't use an `h1`. 
I'm in favor of semantic html where it makes sense but I'm currently not understanding the benefit of transforming an `h1` to look like an `h3` just to have the `h1` present.

Reply:
Just in that link you shared (http://web-accessibility.carnegiemuseums.org/foundations/semantic/) there is a mention: "Use one H1 per page, matching the page title." We didn't do the "matching" part though.

Again, this doesn't really have to do with the designs, as they deal with the aesthetics , not with the document structure. In this case, we are using that "h3" because its content is good enough for our h1. We could also create a hidden h1 as we have done in other pages, but I think this solution is more effective.

Reply:
I had misinterpreted that line as a recommendation to only have one `h1` if you have them, not that there **must** be an `h1`. However, I found a better resource that covers the purpose and explains why it is a must. https://fae.disability.illinois.edu/rulesets/HEADING_1/. I'm all set on this concept.